### PR TITLE
Minor enhancements in readability

### PR DESCRIPTION
**Pull request by StefanSpieker**
I would like to participate in this plugin and I thought I could start by improving some unit tests. So this is just a very simple pull request to kindly ask if improving the unit tests is a good starting point at all.

### CODE REVIEW

File: src/main/resources/hudson/scm/browsers/FishEyeSVN/help-url_de.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div>
   Geben Sie die FishEye-URL des Moduls an, z.B.
-  <tt>http://fisheye6.cenqua.com/browse/ant/<tt>.
+  <tt>http://fisheye6.cenqua.com/browse/ant/</tt>.
```
Comment (oleg-nenashev) [inline src/main/resources/hudson/scm/browsers/FishEyeSVN/help-url_de.html]: oops

**Comment by StefanSpieker** [comment]
I don't understand why the build failed?

**Comment by dwnusbaum** [comment]
This plugin is targeting [Java 7](https://github.com/jenkinsci/subversion-plugin/blob/776e7758f35262696f3b5c3b9bfe270b80c95f09/pom.xml#L139), so the lambdas need to be reverted to anonymous classes (unless the maintainers want to target Java 8).

**Comment by dwnusbaum** [comment]
Looks like #214 will bump java.level to 8.

**Comment by kuisathaverat** [comment]
yep, this Update core to 2.60 so needs java 8

**Comment by oleg-nenashev** [comment]
https://github.com/jenkinsci/subversion-plugin/pull/216

**Pull request merged**